frontend/simple/sass/_groupincome.scss

```diff
@@ -93,3 +93,16 @@
 .is-small {
   font-size: $size-7;
 }
+
+// Have colors variants matching the background
+
+.is-opacity-1 {
```

Comment:
I know these things don't exist in bulma itself but they are still "theme-related" in a way so maybe they could go together with our bulma overwrites. what do you think @sandrina-p ?

Reply:
For clarity purposes, I believe we should keep separated framework classes from our owns. If we merge them all, it might get confused _"Is this from bulma or not?"_ Maybe the location isn't the best at the moment, but my concern were would be to keep them separated.

Reply:
ok, that totally makes sense 👍 then let's keep it here for now